### PR TITLE
Mogu Portraits

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Removed temporary male mogu portraits.
- Added male mogu portraits

### CODE REVIEW

**Comment by sejtalk** [comment]
> Clothes, headgear, and portrait customizations all work fine. Look good too.
> 
> One possible glitch I ran into was that changing ears (the eyebrows) would also change eye color (which was apparently also eyebrows) and vice versa. If this is intended, just let me know and I will approve it. It just seemed like a bug when testing it.

It's a bug but i have zero idea how that happened, why does it work like that or how to fix it so i think that since it's harmless we just have to let it be.

File: interface/portraits/portraits_mogu.gfx [hunk 1]
```diff
@@ -52,10 +74,14 @@ spriteTypes = {
...
-			{ 0 0 0 } {  125 125 125 } { 255 255 255 }	# Default gray
+			{ 0 0 0 } { 196 172 91 } { 255 255 255 } # Blonde
+			{ 0 0 0 } {  75 75 75 } { 255 255 255 } # Black
+			{ 0 0 0 } { 176 155 108 } { 255 255 255 } # Hay
+			{ 0 0 0 } { 125 85 56 } { 255 255 255 } # Brown
+			{ 0 0 0 } { 78 54 37 } { 255 255 255 } # Dark brown
 		}
 
-		eye_color_index = 9
+		eye_color_index = 7
```
Comment (zumbak04) [inline interface/portraits/portraits_mogu.gfx:84]: Index changed.

**Comment by zumbak04** [comment]
Hairs are too bright.
<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by zumbak04** [comment]
This "headgear" looks bad.
It overlays eyebrowns, it's pixelated, it's not clear what it's.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
What it's?
<details>
<summary>Click to expand</summary>

image


</details>

**Comment by sejtalk** [comment]
> This "headgear" looks bad.
> It overlays eyebrowns, it's pixelated, it's not clear what it's.
> 
> Click to expand

So should i delete it?

**Comment by sejtalk** [comment]
> What it's?
> 
> Click to expand
> image
> image

image
It's the end of that thingy that grows from their head

**Comment by zumbak04** [comment]
> So should i delete it?

Or find a replacement.

**Review by zumbak04 — COMMENTED** [review]
🔼

**Comment by sejtalk** [comment]
> > So should i delete it?
> 
> Or find a replacement.

replaced it

**Review by Myrran — APPROVED** [review]
- Customization in ruler designer works fine;
- Barbershop customization works fine;
- Portraits generated in-game work fine.

I like the portraits and customization.

**Comment by zumbak04** [comment]
Forgot to resave.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — COMMENTED** [review]
Forgot to resave.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by sejtalk** [comment]
> Forgot to resave.
> 
> Click to expand

resaved

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**